proofwatch/config.go

```diff
@@ -0,0 +1,45 @@
+package proofwatch
```

Comment:
@jpower432 should this package be renamed to `otelproofwatch`?  I see the guidelines mention "go.opentelemetry.io/contrib/instrumentation/{IMPORT_PATH}/otel{PACKAGE_NAME}".